lib/include/jxl/decode.h

```diff
@@ -963,6 +1012,30 @@ JXL_EXPORT JxlDecoderStatus
 JxlDecoderSetImageOutCallback(JxlDecoder* dec, const JxlPixelFormat* format,
                               JxlImageOutCallback callback, void* opaque);
 
+/** Similar to @c JxlDecoderSetImageOutCallback except that the callback is
+ * allowed an initialization phase during which it is informed of how many
+ * threads will call it concurrently, and those calls are further informed of
+ * which thread they are occurring in.
+ *
+ * @param dec decoder object
+ * @param format format of the pixels. Object owned by user and its contents
```

Comment:
s/and its/whose/

Reply:
Not sure about this one, we might get some opposition: https://en.wikipedia.org/wiki/Inanimate_whose

Reply:
I was more objecting to "and its" rather than suggesting "whose" :P

Reply:
Fair enough :p How is this? I also changed the function from which I copied it in the first place but there are a few more still.